canonical/canonical.com
# Remove the all roles from the department pages

## Done
Removed the all roles tabbed section from the department page as it was failing. This moves the design closer to its destination.

## QA

- Check out the careers overview page
- Click all departments
- Check the department's pages work
- Go to /careers/techops and see that resolves to support engineering
- Submit a test application for your job of choice and check that works

## Issue / Card
Fixes https://github.com/canonical/canonical.com/issues/705
